File: kfcore/src/main/java/com/lq/ren/base/utils/UnitUtils.java
package com.lq.ren.base.utils;

/**
 * Author lqren on 17/10/12.
 */

public class UnitUtils {

    public static class Time {
        public static float ms2min(long ms) {
            return ms / 1000f / 60f;
        }

        public static float s2h(float s) {
            return s / 60f / 60f;
        }
    }

    public static class Distance {
        public static float m2km(float m) {
            return m / 1000f;
        }
    }
}

File: kfcore/src/main/java/com/lq/ren/base/utils/UnitUtils.kt
package com.lq.ren.base.utils

/**
 * Author lqren on 17/10/12.
 */

class UnitUtils {

    object Time {
        fun ms2min(ms: Long): Float {
            return ms.toFloat() / 1000f / 60f
        }

        fun s2h(s: Float): Float {
            return s / 60f / 60f
        }
    }

    object Distance {
        fun m2km(m: Float): Float {
            return m / 1000f
        }
    }
}
